### ISSUE TITLE
Won't start

### ISSUE THREAD

**Issue by VoidNoire** [way-cooler/way-cooler#569]
Trying to start it from TTY results in the error below.

# Minimum issue requirements

**Search the issue tracker first, as it's very possible your question has been asked and answered**

Please provide, at minimum:
* Way Cooler version (obtained via `way-cooler --version`
```
Way Cooler 0.8.0
Way Cooler IPC version 1
```
* rc file
```ion
alias reboot = 'sudo reboot'
alias poweroff = 'sudo poweroff'
alias brightness = 'ion /home/a/scripts/brightness.ion '

alias ls = 'exa -lm --color=auto '
alias l = 'exa -alm --color=auto '
alias la = 'exa -alm --color=auto '
alias lc = 'exa -al --color=always '
alias lac = 'exa -alm --color=always '
alias ld = 'exa -d .* -m --color=auto '
alias ldc = 'exa -d .* -m --color=always '

alias grep = 'rg '
alias find = 'fd '
alias cat = 'bat '
alias c = 'bat '
alias sysupdate = 'ion ~/scripts/update.ion'
alias sysinstall = 'sudo xbps-install -Sv'
alias sysuninstall = 'sudo xbps-remove -Rv'
alias profile = 'micro ~/.profile'
alias notes = 'micro /data/documents/void.txt'
alias hist_dedupe = 'awk '!x[$0]++' ~/.local/share/ion/history > ~/.local/share/ion/history.tmp && mv ~/.local/share/ion/history.tmp ~/.local/share/ion/history'
alias gclone = 'git clone'
alias gpull = 'git pull'
alias gpush = 'git push'
cd ~/

export XKB_DEFAULT_LAYOUT = "gb" # used by way-cooler/wayland
export XDG_RUNTIME_DIR = "/tmp/${UID}-runtime-dir" # used by way-cooler/wayland
if not test -d "${XDG_RUNTIME_DIR}"
	mkdir "${XDG_RUNTIME_DIR}"
	chmod 0700 "${XDG_RUNTIME_DIR}"
end
#export XDG_VTNR = 2 # tty used by way-cooler/wayland
export PATH = "$HOME/bin:$PATH"
export EDITOR = "/usr/bin/micro"
export PAGER = "/usr/bin/less"
export CC = "/usr/bin/clang" # used by cargo
export CXX = "/usr/bin/clang++" # used by cargo

### from rustup, delete this and the following line after uninstall
export PATH = "$HOME/.cargo/bin:$PATH"
```
* Debug log

You can obtain a debug log like so: `way-cooler 2> log.txt`

If the error is a result of a segfault, please also include a backtrace or a coredump.
```
Launching way-cooler...
 INFO [way_cooler] [37msrc/main.rs:215[0m Logger initialized, setting wlc handlers. [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m HOME: /home/a [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m USER: a [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m SHELL: /home/a/.cargo/bin/ion [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m TERM: linux [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m PATH: /home/a/.cargo/bin:/home/a/bin:/usr/local/bin:/bin:/usr/bin [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m MAIL: /var/spool/mail/a [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m LOGNAME: a [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m HOST: a [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m PWD: /home/a [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m OLDPWD: /home/a [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m XKB_DEFAULT_LAYOUT: gb [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m XDG_RUNTIME_DIR: /tmp/1000-runtime-dir [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m EDITOR: /usr/bin/micro [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m PAGER: /usr/bin/less [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m CC: /usr/bin/clang [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m CXX: /usr/bin/clang++ [0m
 INFO [way_cooler] [37msrc/main.rs:220[0m RUST_BACKTRACE: 1 [0m
gbm: failed to open any driver (search paths /usr/lib/xorg/modules/drivers)
gbm: Last dlopen error: /usr/lib/xorg/modules/drivers/radeonsi_dri.so: cannot open shared object file: No such file or directory
failed to load driver: radeonsi
 INFO [wayland] [37msrc/wayland/mod.rs:7[0m Initializing wayland protocols [0m
 INFO [wayland::gamma_control] [37msrc/wayland/gamma_control.rs:233[0m Initializing gamma control manager [0m
 INFO [wayland::desktop_shell] [37msrc/wayland/desktop_shell.rs:170[0m Initializing desktop shell [0m
[37m TRACE [callbacks] [37msrc/callbacks.rs:193[0m[37m Registered wlc callbacks [0m
 INFO [way_cooler] [37msrc/main.rs:271[0m Running wlc... [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: libinput: resume [0m
thread '<unnamed>' panicked at 'Unable to create dbus session: D-Bus error: Unable to autolaunch a dbus-daemon without a $DISPLAY for X11 (org.freedesktop.DBus.Error.NotSupported)', src/libcore/result.rs:906:4
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: MODE: (0) 1366x768@60000 * [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: MODE: (0) 1366x768@40000  [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: MODE: (0) 1280x720@60000  [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: MODE: (0) 1152x768@60000  [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: MODE: (0) 1024x768@60000  [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: MODE: (0) 800x600@60000  [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: MODE: (0) 848x480@60000  [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: MODE: (0) 720x480@60000  [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: MODE: (0) 640x480@60000  [0m
[33m WARN [way_cooler] [37msrc/main.rs:66[0m[33m wlc: Connector 1 is not connected or has no modes [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: eDP-1 Chose mode (0) 1366x768 [0m
[37m TRACE [callbacks] [37msrc/callbacks.rs:37[0m[37m output_resolution: WlcOutput { handle: 1, name: "eDP-1", views: [] } from  Size { w: 0, h: 0 } to Size { w: 1366, h: 768 } [0m
[33m WARN [layout::actions::layout] [37msrc/layout/actions/layout.rs:521[0m[33m LayoutTree {
    tree: InnerTree {
        graph: StableGraph {
            Ty: "Directed",
            node_count: 1,
            edge_count: 0,
            node weights: {
                0: Root(
                    Uuid("003bba02-99f0-42b8-8bff-62fd020f9e86")
                )
            },
            edge weights: {},
            free_node: NodeIndex(4294967295),
            free_edge: EdgeIndex(4294967295)
        },
        id_map: {},
        view_map: {},
        root: NodeIndex(0),
        active_path: "0"
    },
    active_container: None
} did not have a parent of type Output, doing nothing! [0m
[37m TRACE [callbacks] [37msrc/callbacks.rs:18[0m[37m output_created: WlcOutput { handle: 1, name: "eDP-1", views: [] }: eDP-1 [0m
[37m TRACE [layout::core::tree] [37msrc/layout/core/tree.rs:453[0m[37m Adding new output with WlcOutput { handle: 1, name: "eDP-1", views: [] } [0m
[37m TRACE [layout::core::graph_tree] [37msrc/layout/core/graph_tree.rs:508[0m[37m Normalized edge weights for: NodeIndex(0) [0m
 INFO [layout::core::graph_tree] [37msrc/layout/core/graph_tree.rs:240[0m Added new child NodeIndex(1) for Output { handle: WlcOutput { handle: 1, name: "eDP-1", views: [] }, background: None, bar: None, id: Uuid("705b27e2-b671-45d7-9d08-863966bd0ed3") } [0m
[37m TRACE [layout::actions::workspace] [37msrc/layout/actions/workspace.rs:43[0m[37m Adding workspace Workspace { name: "1", geometry: Geometry { origin: Point { x: 0, y: 0 }, size: Size { w: 1366, h: 768 } }, fullscreen_c: [], id: Uuid("e6b0ae89-3275-4a4d-bfa2-34c09e4793f0") } [0m
[37m TRACE [layout::core::graph_tree] [37msrc/layout/core/graph_tree.rs:508[0m[37m Normalized edge weights for: NodeIndex(1) [0m
 INFO [layout::core::graph_tree] [37msrc/layout/core/graph_tree.rs:240[0m Added new child NodeIndex(2) for Workspace { name: "1", geometry: Geometry { origin: Point { x: 0, y: 0 }, size: Size { w: 1366, h: 768 } }, fullscreen_c: [], id: Uuid("e6b0ae89-3275-4a4d-bfa2-34c09e4793f0") } [0m
[37m TRACE [layout::core::graph_tree] [37msrc/layout/core/graph_tree.rs:508[0m[37m Normalized edge weights for: NodeIndex(2) [0m
 INFO [layout::core::graph_tree] [37msrc/layout/core/graph_tree.rs:240[0m Added new child NodeIndex(3) for Container { layout: Horizontal, floating: false, fullscreen: false, output_handle: WlcOutput { handle: 1, name: "eDP-1", views: [] }, apparent_geometry: Geometry { origin: Point { x: 0, y: 0 }, size: Size { w: 1366, h: 768 } }, geometry: Geometry { origin: Point { x: 0, y: 0 }, size: Size { w: 1366, h: 768 } }, id: Uuid("ef01e75d-656f-43f7-a54d-4df457ca1c76"), borders: None } [0m
[44m DEBUG [layout::commands] [37msrc/layout/commands.rs:416[0m[44m Layout.SwitchWorkspace("1") [0m
[37m TRACE [layout::actions::workspace] [37msrc/layout/actions/workspace.rs:78[0m[37m Switching to workspace 1 [0m
[33m WARN [way_cooler] [37msrc/main.rs:66[0m[33m wlc: Arguments are inconsistent [0m
[33m WARN [way_cooler] [37msrc/main.rs:66[0m[33m wlc: Could not initialize any context [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: Released drm surface (0x7f6aba473160) [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: Removed bsurface from output (1) [0m
[37m TRACE [callbacks] [37msrc/callbacks.rs:30[0m[37m output_focus: WlcOutput { handle: 1, name: "eDP-1", views: [] } focus=true [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: Added output (1) [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: become active [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: Cleanup wlc [0m
 INFO [callbacks] [37msrc/callbacks.rs:140[0m Preparing compositor! [0m
 INFO [lua::thread] [37msrc/lua/thread.rs:210[0m Running lua on_init() [0m
 INFO [lua::thread] [37msrc/lua/thread.rs:122[0m Initializing lua... [0m
 INFO [lua::thread] [37msrc/lua/thread.rs:127[0m Loading way-cooler libraries... [0m
[37m TRACE [lua::rust_interop] [37msrc/lua/rust_interop.rs:22[0m[37m Registering Rust libraries... [0m
[37m TRACE [lua::rust_interop] [37msrc/lua/rust_interop.rs:46[0m[37m Executing Lua init... [0m
[31m ERROR [awesome] [37msrc/awesome/mod.rs:94[0m[31m xcb: Could not connect to xwayland instance. Is it running? [0m
[31m ERROR [awesome] [37msrc/awesome/mod.rs:95[0m[31m ClosedParseErr [0m
[37m TRACE [lua::rust_interop] [37msrc/lua/rust_interop.rs:55[0m[37m Lua register_libraries complete [0m
[37m TRACE [lua::init_path] [37msrc/lua/init_path.rs:65[0m[37m Looking for init file at ~/.config/way-cooler/init.lua [0m
[33m WARN [lua::init_path] [37msrc/lua/init_path.rs:70[0m[33m No init file found in ~/.config, will default to /etc as a last resort. [0m
[33m WARN [lua::init_path] [37msrc/lua/init_path.rs:81[0m[33m way-cooler was unable to find an init file. Using our default, pre-compiled init.lua. [0m
[33m WARN [lua::init_path] [37msrc/lua/init_path.rs:83[0m[33m Our default init.lua is available on GitHub, it is included with the source. [0m
[33m WARN [lua::init_path] [37msrc/lua/init_path.rs:85[0m[33m You may place it in ~/.config/way-cooler/init.lua, or a similar folder if you use $XDG_CONFIG_HOME, or /etc/way-cooler. [0m
[33m WARN [lua::init_path] [37msrc/lua/init_path.rs:87[0m[33m You may also use the environment variable $WAY_COOLER_INIT_FILE to point directly to the file of your choice. [0m
[33m WARN [lua::thread] [37msrc/lua/thread.rs:173[0m[33m Defaulting to pre-compiled init.lua [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("d")) for Event: KeyEvent::Lua, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("Return")) for Event: KeyEvent::Lua, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("l")) for Event: KeyEvent::Lua, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("h")) for Event: KeyEvent::Lua, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("l")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("Left")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("Right")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("Up")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("Down")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("Left")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("Right")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("Up")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("Down")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("h")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("v")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("e")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("s")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("w")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("f")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("q")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("space")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("space")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("r")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("1")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("1")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("2")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("2")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("3")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("3")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("4")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("4")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("5")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("5")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("6")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("6")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("7")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("7")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("8")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("8")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_ALT, Some("9")) for Event: KeyEvent::Command, passthrough: false  [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("9")) for Event: KeyEvent::Command, passthrough: false  [0m
 INFO [lua::thread] [37msrc/lua/thread.rs:186[0m Entering main loop... [0m
[44m DEBUG [layout::commands] [37msrc/layout/commands.rs:640[0m[44m Layout.focus("ef01e75d-656f-43f7-a54d-4df457ca1c76") [0m
[44m DEBUG [layout::commands] [37msrc/layout/commands.rs:590[0m[44m Layout.Focus("ef01e75d-656f-43f7-a54d-4df457ca1c76") [0m
 INFO [layout::core::tree] [37msrc/layout/core/tree.rs:231[0m Active container was 3, is now 3 [0m
[37m TRACE [keys] [37msrc/keys/mod.rs:88[0m[37m Registering Keypress(MOD_SHIFT | MOD_ALT, Some("Escape")) for Event: KeyEvent::Command, passthrough: false  [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: Closing Xwayland [0m
 INFO [way_cooler] [37msrc/main.rs:65[0m wlc: Closed drm [0m
[37m TRACE [lua::thread] [37msrc/lua/thread.rs:226[0m[37m Lua: awaiting request [0m
[37m TRACE [lua::thread] [37msrc/lua/thread.rs:238[0m[37m Handling a request [0m
[37m TRACE [lua::thread] [37msrc/lua/thread.rs:399[0m[37m Updating programs:String(LuaRef(12)) = String(
    "lemonbar"
) [0m
[37m TRACE [lua::thread] [37msrc/lua/thread.rs:399[0m[37m Updating mouse:String(LuaRef(8)) = Boolean(
    false
) [0m
stack backtrace:
[37m TRACE [lua::thread] [37msrc/lua/thread.rs:226[0m[37m Lua: awaiting request [0m
[37m TRACE [lua::thread] [37msrc/lua/thread.rs:238[0m[37m Handling a request [0m
[37m TRACE [lua::thread] [37msrc/lua/thread.rs:285[0m[37m Received request to execute way_cooler.on_init() [0m
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: <unknown>
   5: <unknown>
   6: <unknown>
   7: <unknown>
   8: <unknown>
   9: <unknown>
  10: <unknown>
  11: <unknown>
  12: <unknown>
  13: <unknown>
  14: <unknown>
  15: <unknown>
  16: start_thread
[31m ERROR [way_cooler] [37msrc/main.rs:67[0m[31m wlc: Failed to activate vt7 for restoration [0m
[31m ERROR [way_cooler] [37msrc/main.rs:67[0m[31m wlc: Failed to switch back to vt1 [0m
sh: 1: wc-bg: not found

```
# Bonus information
If you provide this information as well, I can respond to your request much faster:
* Is the issue reproducible on Sway?
  - We share the same backend, so if it's not a Way Cooler specific issue it might affect them too
I haven't installed Sway.
* Is the issue reproducible on Awesome?
  - It may not be a bug, it could be a feature.
I haven't installed Awesome.
* What distro do you run?
Void Linux
* How did you install Way Cooler?
  - E.g: build from source? Package manager? Found on a disk by the side of the road?
Basically with `xbps-install -S way-cooler`.

# Awesome specific questions
If the bug report is an issue with the Awesome API, please mention the appropriate method/object from the Awesome Lua API and describe how it is not working as intended.

**Comment by VoidNoire** [comment]
Ah no worries.

I don't actually think I have a Way Cooler Rc file (which one is it supposed to be again? I couldn't find any documentation or man pages about it unfortunately). But yeah, I think it's definitely a good idea to clarify which file is required in the issue template.

I've since installed the X11 window manager `bspwm` and `sway` (not sure if it uses wlc, but `sway --version` indicates that it's `sway version  (2018-04-18, branch "")`). I've found out that both `way-cooler` and `sway` are unable to be launched from the TTY (`way-cooler` crashes with the message above, and `sway` freezes the screen), but both can be launched from `bspwm`. Maybe this is related/the same issue as #351?
